cmd/flags/auth.go

```diff
@@ -108,26 +154,36 @@ Environment variable: %q`, authOIDCCACertFileEnv)
 }
 
 func NewAuthConfig(vipr *viper.Viper) (*auth.Config, error) {
-	certs := []*x509.Certificate{}
+	// OIDC part
+	certsOIDC := []*x509.Certificate{}
 
-	fileCert, err := fileCertificate(vipr)
+	fileCertOIDC, err := fileCertificate(vipr)
 	if err != nil {
 		return nil, err
-	} else if fileCert != nil {
-		certs = append(certs, fileCert)
+	} else if fileCertOIDC != nil {
+		certsOIDC = append(certsOIDC, fileCertOIDC)
 	}
 
 	issuerCerts, err := issuerCertificates(vipr)
 	if err != nil {
 		return nil, err
 	} else if issuerCerts != nil {
-		certs = append(certs, issuerCerts...)
+		certsOIDC = append(certsOIDC, issuerCerts...)
```

Comment:
Minor: Still doing append when it is not needed

Reply:
This part is the OIDC one, seems needs